apps/melding-form/src/app/(general)/Home.test.tsx

```diff
@@ -35,4 +36,37 @@ describe('Page', () => {
 
     expect(screen.queryByText('Test error message')).toBeInTheDocument()
   })
+
+  it('should fetch defaultValues from localStorage', async () => {
```

Comment:
I don't think this needs to be `async`. Same goes for this test on other pages. 

```suggestion
  it('should fetch defaultValues from localStorage', () => {
```